BenchmarkTool/Config.cs

```diff
@@ -31,14 +81,22 @@ public static int GetTestRetries()
             int.TryParse(val, out int loop);
             return loop;
         }
-
+        private static string _DbSetting = "null";
         public static string GetTargetDatabase()
         {
             var val = ConfigurationManager.AppSettings[ConfigurationKeys.TargetDatabase];
+            if (_DbSetting.Contains("DB"))
+            {
+                val = _DbSetting;
+            }
             if (String.IsNullOrEmpty(val))
                 throw new Exception(String.Format("Null or empty app settings val for key={0}", ConfigurationKeys.TargetDatabase));
             return val;
         }
+        internal static void SetTargetDatabase(string setting)
```

Comment:
As stated above, this would break current implementation of CLI args . In Addition, keeping this,in future it could be easily implemented in the BM-loop, to test an array of databases in a row -similar to clientsnumbers, bachsize, and so on.